astropy/time/tests/test_precision.py

```diff
@@ -391,6 +399,11 @@ def test_resolution_never_decreases(scale, jds):
 @given(reasonable_jd())
 def test_resolution_never_decreases_utc(jds):
     """UTC is very unhappy with unreasonable times"""
+
+    # FIXME: This fails because t == t + dt_tiny
+    if jds[1] == 0.9999999999999999:
```

Comment:
Do you have an explicit example for when this fails? May be related to the same bug in `day_frac`

Reply:
Not sure. I can remove it so you see the failure.